gulpfile.js

```diff
@@ -155,6 +155,29 @@ gulp.task('browserify', ['jison'], function () {
 });
 
 /**
+ * Task definition for bundling up the project files into a single JS file.
+ */
+gulp.task('browserify-layout-testing', ['jison'], function () {
+    'use strict';
```

Comment:
Well this whole file is formatted differently. I'm being consistent with rest of the file.